pkg/minikube/notify/notify.go

```diff
@@ -99,16 +100,23 @@ func getJson(url string, target *releases) error {
 }
 
 func getLatestVersionFromURL(url string) (semver.Version, error) {
```

Comment:
Yeah i see that now.  Weird, I could have sworn that I encountered this a few days ago on my personal workstation